targets/tofino/base/expression_resolver.cpp

```diff
@@ -440,6 +440,8 @@ const ExternMethodImpls EXTERN_METHOD_IMPLS(
           BUG_CHECK(components.size() == 1, "Expected 1 component, got %1%", components.size());
           const auto *applyDecl = components.at(0)->checkedTo<IR::Function>();
           const auto *applyParameters = applyDecl->type->parameters;
+          BUG_CHECK(applyParameters->size() == 1 || applyParameters->size() == 2,
+                    "Expected 1 or 2 parameters, got %1%", applyParameters->size());
```

Comment:
For the `apply` function with only one parameter, it doesn't return anything, can we just return nullptr if `applyParameters->size() == 1`?